.gitignore

```diff
@@ -49,3 +49,10 @@ _dist/
 # Distribution storage files for local E2E testing
 test/e2e/testdata/distribution/mount/docker/
 test/e2e/testdata/distribution/mount_fallback/docker/
+
```

Comment:
This seems like a pain to maintain, not that I have a better idea.

Reply:
Future changes will only be required when adding or removing new repositories. E.g. If I add a new image repository for `new-test` suite, I only need to add

> !test/e2e/testdata/zot/new-test/images

Looks like I need to cover that part in the `test/e2e/README.md`

Reply:
Added test data baking guide.